static_src/components/users.jsx

```diff
@@ -24,6 +24,14 @@ const ORG_MANAGER = 'org_manager';
 const SPACE_MANAGER = 'space_manager';
 const ORG_ENTITY = 'organization';
 const SPACE_ENTITY = 'space';
+const ORG_INVITE_HELP = 'Only an Org Manager can new invite users to this ' +
```

Comment:
@rememberlenny I actually have a ticket somewhere in the backlog to start rounding up strings and splitting them into constant files! 😄